tests/foreman/ui_airgun/test_contenthost.py

```diff
@@ -515,3 +554,250 @@ def test_positive_virt_who_hypervisor_subscription_status(session):
                 virt_who_hypervisor_host['name'])[0]['Subscription Status'] == 'green'
             chost = session.contenthost.read(virt_who_hypervisor_host['name'])
             assert chost['details']['subscription_status'] == 'Fully entitled'
+
+
+@tier3
+def test_module_stream_actions_on_content_host(session, vm_module_streams):
```

Comment:
Supported actions are: 'Enable', 'Disable', 'Install', 'Update', 'Remove' and 'Reset'. This test covers 'Install', 'Remove', 'Disable' and 'Reset', i.e. it doesn't cover 'Enable' and 'Update'.

Could it be changed to also cover these two actions? Could they be covered in new tests?

Reply:
Update only comes, in place of errata. I will add a new test case for this, not wanted everything in one test case and it will be too long.

Reply:
Missing actions added in `test_module_stream_update_from_satellite`